Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM java:openjdk-8-alpine
+
+COPY ./target/cqlTranslationServer-1.0-SNAPSHOT-jar-with-dependencies.jar /app/
```

Comment:
I think it would be better if the Docker file builds the service rather than just copying the pre-built uberjar.